src/scheduler/workers/processListsBeforeAndDuringStart/helpers.ts

```diff
@@ -14,36 +12,114 @@ export function formatDate(date: Date = todayDateString) {
+export function isEmailSentBefore(event: Audit | undefined, reminderType: RemindersBeforeStartDate): boolean {
+  if (!event?.jsonData || !reminderType || !event) {
     return false;
   }
-  const subsequentEmailsForReminderType = {
-    sendOneMonthPostEmail: [
-      "sendOneMonthPostEmail",
-      "sendOneWeekPostEmail",
-      "sendOneDayPostEmail",
-      "sendStartPostEmail",
-    ],
-    sendOneWeekPostEmail: ["sendOneWeekPostEmail", "sendOneDayPostEmail", "sendStartedPostEmail"],
-    sendOneDayPostEmail: ["sendOneDayPostEmail", "sendStartedPostEmail"],
-    sendStartedPostEmail: ["sendStartedPostEmail"],
-    sendStartedProviderEmail: ["sendStartedProviderEmail"],
+
+  const subsequentEmailsForReminderType: Record<RemindersBeforeStartDate, ListAnnualReviewPostReminderType[]> = {
+    oneMonthBeforeStart: ["oneMonthBeforeStart", "oneWeekBeforeStart", "oneDayBeforeStart", "started"],
+    oneWeekBeforeStart: ["oneWeekBeforeStart", "oneDayBeforeStart", "started"],
+    oneDayBeforeStart: ["oneDayBeforeStart", "started"],
+    started: ["started"],
   };
+
   const subsequentEmails: string[] = subsequentEmailsForReminderType[reminderType];
-  let reminderHasBeenSent = false;
-  if ("createdAt" in event) {
-    reminderHasBeenSent = subsequentEmails.includes((event.jsonData as ListEventJsonData)?.reminderType as string);
-  } else {
-    reminderHasBeenSent = subsequentEmails.includes((event.jsonData as Record<string, string>)?.notes[0]);
-  }
+  const reminderHasBeenSent = subsequentEmails.includes(
+    event.annualReviewEmailType ?? (event.jsonData as Record<string, string>)?.reminderType
+  );
 
   if (reminderHasBeenSent) {
     logger.info(`Email has been sent before for ${reminderType} reminder type`);
     return true;
   }
   return false;
 }
+
+/**
+ * if `emailType` is `oneMonthBeforeStart` and an email has been sent for `oneWeekBeforeStart` or `oneDayBeforeStart` or `started`, this function will return false.
+ * @param emailType the `{@link ListAnnualReviewPostReminderType}` to check. If an email has been sent for this type OR a type larger than this type, this function will return false.
+ * @param reference annual review reference
+ */
+export async function shouldSendToPost(emailType: ListAnnualReviewPostReminderType, reference: string) {
+  /**
+   * This query uses postgres enums. Enums in postgres are ordered so `>=` operator can be used.
+   * Looks for events with `listItemId` and where an annualReviewEmailType > than `emailType` has occurred.
+   */
+  const query: Prisma.PrismaPromise<Audit[] | undefined> = prisma.$queryRaw`select * from "Audit"
+           where "annualReviewEmailType" >= ${emailType}::"AnnualReviewPostEmailType"
+           and "jsonData"->>'annualReviewRef' = '${reference}'
+         order by "createdAt" desc limit 1`;
+
+  let result: Audit[] | undefined;
+
+  try {
+    result = await query;
```

Comment:
Out of curiosity what's the benefit of assigning the query to a variable instead of just running it here?